docs/getting-started/getting-started-chunks.md

```diff
@@ -21,6 +21,8 @@ Cortex can also make use of external memcacheds for caching and although these a
 For simplicity and to get started, we'll run it as a [single process](../configuration/single-process-config.md) with no dependencies:
 
 ```sh
+$ git clone git@github.com:cortexproject/cortex.git
```

Comment:
I would suggest to use HTTP-based clone instead, as it doesn't depend on `ssh` and can use proxies.

```suggestion
$ git clone https://github.com/cortexproject/cortex.git
```

Reply:
That make sense. Fixing.